src/utils/commons.ts

```diff
@@ -112,7 +112,7 @@ export function getOrCreateZTokenStatus(holder: Address): ZTokenStatus {
 }
 
 export function updateZTokenBalancesFor(
-  zToken: string,
+  platformToken: string,
```

Comment:
Just a comment. Given that we have tokens for lenders(TDAI, TUSDC), and tokens for governance (ATMTokens) I find platformToken a bit unclear, I would need first to understand the context to know which token we are referring to here. I would use ttoken for lending and atmtoken or govtoken for governance.

Reply:
As we want to rename/change the zToken (now they are tToken), I renamed them to platform tokens (more generic).

The platform token would be the tTokens.